obs_github_deployments.gemspec

```diff
@@ -33,6 +33,8 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rake", "~> 13.0"
   spec.add_development_dependency "rspec", "~> 3.0"
   spec.add_development_dependency "rubocop", "~> 1.7"
+  spec.add_development_dependency "rubocop-rake", "~> 0.5"
+  spec.add_development_dependency "rubocop-rspec", "~> 2.2"
```

Comment:
@saraycp I think you need to tell rubcop to use the extension https://github.com/rubocop-hq/rubocop-rspec#rubocop-configuration-file

Reply:
Of course!